api/users/usersRepo.js

```diff
@@ -99,6 +99,31 @@ module.exports.factory = (db, User, { Blueprint, is }) => {
     })
   }
 
+  /*
+// Add a purchase to purchase history
+*/
+  const getPurchases = (email) => {
+    return new Promise((resolve, reject) => {
+      if (is.not.string(email)) {
+        return reject(new Error('An email is required to get purchase history'))
+      }
+
+      collection.find({ email })
+        .limit(1)
+        .next((err, doc) => {
+          if (err) {
+            return reject(err)
+          }
+
+          if (doc) {
+            return resolve(new UserPurchaseHistory(doc))
+          } else {
+            return resolve()
```

Comment:
The else should be return reject() right?

Reply:
👍